scripts/get_defcore_tests.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Is this file needed any more now that the tasks are merged into run_tempest.yml ?